tests/Resources/ti.ui.window.test.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-Present by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* global Ti */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+
+var should = require('./utilities/assertions'),
+	utilities = require('./utilities/utilities');
+
+describe('Titanium.UI.Window', function () {
```

Comment:
rename this file to `ti.ui.window.navigationwindow.addontest.js` so it gets "added on" the existing suite rather than replaces the existing ti.ui.window tests.

Reply:
Alternatively, you could add a `ti.ui.ios.navigationwindow.addontest.js` for this create/property test and any future tests on that type (I see we have 3 methods that likely should have tests: http://docs.appcelerator.com/platform/latest/#!/api/Titanium.UI.iOS.NavigationWindow)

Reply:
How to deal with `before`, `after`, `beforeEach` and `afterEach` methods? Should I include those in the addons?

Reply:
Looks like to be safe on Android the before/after should stay. You don't check for, so don't need `didFocus` or `before` hook. I'd keep the `after` hook, and drop your explicit setting of `win = null` in the test (is that needed?)

Reply:
Okay, will do. Copied over the `win = null` statement, don't think it's needed as the proxy should clean up after itself.

Reply:
Decided to use `ti.ui.ios.navigationwindow.addontest.js` and will add unit-tests for the other ones as well to get some more basic coverage. And as Android / Windows will get parity on the navigation-window, they'll be happy to find existing unit-tests already.